cmd/stack_package.go

```diff
@@ -195,7 +197,16 @@ The packaging process builds the stack image, generates the "tar.gz" archive fil
 
 			// docker build
 			// create the image name to be used for the docker image
-			namespaceAndRepo := imageNamespace + "/" + stackID
+			imageRepo := rootConfig.CliConfig.GetString("images")
+
+			if imageRegistry != "dev.local" && imageNamespace == "appsody" {
```

Comment:
I don't think we need any of this logic anymore. Just make 
`namespaceAndRepo = imageRegistry + "/" + imageNamespace + "/" + stackID`

Now that we've moved `dev.local` to the registry, we don't need to read "images" from the CliConfig. Just always use the value in the flag.

Reply:
My intention was to use "images" if they set a namespace but not registry, and only use `dev.local` if neither the namespace nor registry was set. If I don't use "images" here, where else is it being used/why do we need it?

Reply:
Thanks for removing this. 
"images" from `~/.appsody/.appsody.yaml` is being used elsewhere. Specifically when pulling a stack that was not fully qualified during `appsody run`